dockerfiles/Dockerfile.training

```diff
@@ -21,11 +21,14 @@ WORKDIR /stage
 
 # install curl and git
 RUN apt-get -y update &&\
+    apt-get install -y sudo git bash unattended-upgrades && \
```

Comment:
you already install git below in line 27?
do you need the other packages that previously weren't being installed? 
can just add unattended-upgrades to the packages below

Reply:
Just added unattened-upgrades now